.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+    
+    services:
+      redisgraph:
```

Comment:
I guess the right name for the service should be `redislabs-redistimeseries` otherwise tests are not gonna work

Reply:
It's just a service name doesn't have a real meaning but you're right it's a C&P error, I renamed it to redistimeseries

Reply:
What I meant is that the host for tests is configured as `redislabs-redistimeseries` as you can see [here](https://github.com/averias/redis-time-series/blob/master/src/__tests_config__/data.ts#L4).

I'm a newbie in GiHub actions but it seems that the service name can be used as hostname when we run the job in a container as specified [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-service-containers#communicating-with-service-containers), but I think that with the current configuration we are running the job in the runner machine so the host only can be `localhost` or `127.0.0.1`

I'm probably wrong since as I said it is the first time I'm using GitHub Actions, anyway if you consider that the tests will run having the hostname pointing to `redislabs-redistimeseries` and without any change in your ci.yml file, let me know and I'll approve this PR and we can go ahead.

Reply:
I renamed the hostname to localhost but it didn't help the tests pass

Reply:
I changed the host to localhost and it now seems to fail on other issues

Reply:
https://github.com/gkorland/redis-time-series/runs/412464789?check_suite_focus=true
```
Test Suites: 4 failed, 20 passed, 24 total
Tests:       4 failed, 155 passed, 159 total
```

Reply:
Since it seems that it is not clear how passing tests with github actions, I tried Travis and it seems to work, you can see in this [PR](https://github.com/averias/redis-time-series/pull/7). I still have to make CodeClimate receives the report and solve some code smells before adding badgets. Now I'm busy but I will do it during the day.

Thank you so much for your time effort, I really appreciate it :)

I would have liked to have CI with GH Actions in this project and learn it a little bit but, at least in my case, I have no so much time now